test/test_quantized.py

```diff
@@ -467,46 +521,39 @@ def test_qconv(
 
         b_init = torch.from_numpy(np.random.randint(0, 10, (output_channels,)))
 
-        # Existing floating point conv operator
-        conv_op = torch.nn.Conv2d(
-            input_channels,
-            output_channels,
-            (kernel_h, kernel_w),
-            (stride_h, stride_w),
-            (pad_h, pad_w),
-            (dilation_h, dilation_w),
-            groups,
-        )
-
-        # assign the weights
-        conv_op.weight = torch.nn.Parameter(
-            W_init.to(dtype=torch.float), requires_grad=False
-        )
-        conv_op.bias = torch.nn.Parameter(
-            b_init.to(dtype=torch.float), requires_grad=False
-        ) if use_bias else None
+        stride = [stride_h, stride_w]
+        pad = [pad_h, pad_w]
+        dilation = [dilation_h, dilation_w]
 
         X_value_min = 0
         X_value_max = 4
         X_init = torch.from_numpy(np.random.randint(
             X_value_min, X_value_max, (batch_size, input_channels, height, width)))
 
-        # run on an input tensor
-        result_ref = conv_op(X_init.to(dtype=torch.float))
+        X_scale = 1.5
```

Comment:
Can you add randomized testing for scale, zero-point values, so that we get more coverage?

Reply:
I am already getting time outs with the number of parameters I have.

Reply:
@raghuramank100: Current version of hypothesis (in internal code) doesn't allow me to increase deadline. In fact, I have to reduce the space as some of the tests are failing in OSS tests. Will update with reduced parameters.

Reply:
Testing for scale and zero_points is important, IMHO. I honestly don't think that it's the hypothesis that is taking very long, but the test itself